lib/compress/zstd_compress.c

```diff
@@ -623,6 +644,14 @@ static size_t ZSTD_resetCCtx_internal(ZSTD_CCtx* zc,
             return ZSTD_continueCCtx(zc, params, pledgedSrcSize);
     }   }
 
+    {
+        zc->ldmState.hashLog = LDM_HASH_LOG;
+        zc->ldmState.bucketLog =
+            MIN(LDM_BUCKET_SIZE_LOG, LDM_BUCKET_SIZE_LOG_MAX);
+        zc->ldmState.hashEveryLog =
```

Comment:
What if `cParams.windowLog < ldmState.hashLog` (caught by the simple_round_trip fuzzer with UBSAN)?